controllers/admin.js

```diff
@@ -381,7 +428,7 @@ exports.apiAdminUpdate = function (aReq, aRes, aNext) {
   });
 
   Strategy.find({}, function (aErr, aStrats) {
-    var stored = nil();
+    var stored = nil(); // NOTE: Inconsistent usage of object creation here
```

Comment:
@sizzlemctwizzle Cc: @Zren 

Again... Is `nil` helper really needed here? So far I haven't seen any Object prototype lines in the controllers... but I haven't looked everywhere.